website/sidebars.js

```diff
@@ -18,6 +18,7 @@ const sidebars = {
         type: 'generated-index',
       },
       collapsed: false,
+      collapsible: false,
```

Comment:
Let's disable collapsing for this category as a dogfooding approach.

Reply:
We have another section to dogfood now, maybe you can move these tests there? 

(one with/without link and with/without collapsible?)

https://deploy-preview-6622--docusaurus-2.netlify.app/tests/docs

Reply:
Oh yes, that's right, I forgot about that. I guess it's okay if I disable collapsing for the category "HTML items tests".